gemfirexd/core/src/main/java/com/pivotal/gemfirexd/internal/impl/sql/execute/TablePrivilegeInfo.java

```diff
@@ -226,8 +226,16 @@ public void executeGrantRevoke( Activation activation,
 									List grantees)
 		throws StandardException
 	{
-		doExecuteGrantRevoke(activation, grant, grantees, columnBitSets, actionAllowed, true, td);
+
 		GemFireStore ms = Misc.getMemStore();
+		if (ms.isSnappyStore() && Misc.isSecurityEnabled() &&
```

Comment:
nitpick: Can use a boolean variable for first two checks which are again done at line#239 and use it at both places.

Reply:
will do that. thanks